pages/feedback.js

```diff
@@ -93,17 +102,24 @@ const SettingsPage = () => {
       setShowError(true)
     }
 
-    setSumitting(false)
+    setSubmitting(false)
   }
 
   return (
     <Layout>
       <div className='flex flex-col justify-center items-center w-full h-full'>
         <div className='flex justify-center items-center w-full h-full px-4 lg:w-132 lg:p-0'>
           <Paper className='flex flex-col h-auto w-full p-4 my-2'>
-            <h3 className='text-xl text-gray-600 font-semibold mx-8 mb-0'>
-              Submit a Bug Report or Feedback
-            </h3>
+            <div  className='flex flex-row'>
+              <h3 className='flex-auto text-xl text-gray-600 font-semibold mx-8 mb-0'>
+                Submit a Bug Report or Feedback
+              </h3>
+              <CloseIcon
+                id='settings_card_close'
+                className={`${classes.pointerIcon} flex-none mt-4 mr-7 mb-0`}
```

Comment:
Great job using Tailwindcss, should we use the `cursor-pointer` class instead to make it consistent with the rest of the classes?
https://tailwindcss.com/docs/cursor#pointer